apps/site/components/Common/Search/index.module.css

```diff
@@ -1,6 +1,5 @@
 .searchButton {
   @apply flex
-    w-52
```

Comment:
what happens if we use `w-fit` ?

Reply:
It is a bad practice explicitly to define a `width` to a content element because it should be defined by using padding.